### PR TITLE
Update magic.erl

### PR DESCRIPTION
**Pull request by elbrujohalcon**

### CODE REVIEW

File: src/magic.erl [hunk 1]
```diff
@@ -1,3 +1,4 @@
+%% @doc this comment is really really really too long, you should reject this PR, @elvisci. Please, oh please. Reject it, I tell you.
```
Comment (elvisci) [inline src/magic.erl]: Line 1 is too long: %% @doc this comment is really really really too long, you should reject this PR, @elvisci. Please, oh please. Reject it, I tell you..

Comment (gadgetci) [inline src/magic.erl]: According to **Elvis**:
> Line 1 is too long: %% @doc this comment is really really really too long, you should reject this PR, @elvisci. Please, oh please. Reject it, I tell you..

Comment (elvisci) [inline src/magic.erl]: Line 1 is too long: %% @doc this comment is really really really too long, you should reject this PR, @elvisci. Please, oh please. Reject it, I tell you..

Comment (gadgetci) [inline src/magic.erl]: According to **Elvis**:

> Line 1 is too long: %% @doc this comment is really really really too long, you should reject this PR, @elvisci. Please, oh please. Reject it, I tell you..

**Pull request merged**